libs/community/langchain_community/document_loaders/mintbase.py

```diff
@@ -0,0 +1,188 @@
+import os
+import re
+import time
+from enum import Enum
+from typing import List, Optional
+
+import requests
+import json
+from langchain_core.documents import Document
+
+from langchain_community.document_loaders.base import BaseLoader
+
+
+class BlockchainType(Enum):
+    """Enumerator of the supported blockchains."""
+
+    NEAR_MAINNET = "mainnet"
+    NEAR_TESTNET = "testnet"
+
+
+class MintbaseDocumentLoader(BaseLoader):
```

Comment:
Could you follow the example here. We want lazy_load implemented rather than load in all new implementations


https://python.langchain.com/docs/modules/data_connection/document_loaders/custom/#implementation